### PR TITLE
Document "racking model" for temperature models

### LINKED ISSUE #1323: Missing explanation of mounting type in document
**Describe the bug**

pvlib.temperature.sapm_cell and pvlib.temperature.sapm_module can be specified coefficient a and b.
and It's depends on module and mounting.
In document, mounting method is listed, but is not described detail.

On the other hand, mounting method detail was described in NREL SAM Help System, and also mentioned in NREL SAM forum.
help system code: https://github.com/NREL/SAM/blob/master/deploy/runtime/help/html/pv_module.htm
forum: https://sam.nrel.gov/forum/forum-general/1040-mount-types.html?start=0

This is missing of mounting type explanation in the document.

**To Reproduce**

**Expected behavior**

Explanations of mounting types are described in the document.

**Screenshots**

**Versions:**

0.9.0

**Additional context**

(if this issue have to be a Support question, I re-ask in  StackOverflow)

### PR DESCRIPTION
**Pull request by cwhanse**
- [x] Closes #1323
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - ~~[ ] Tests added~~
 - ~~[ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.~~
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

Also removes the no-longer correct default from the docstring of `PVSystem`

### CODE REVIEW

**Comment by RDaxini** [comment]
I think we have some overlap here with #2232

**Comment by cwhanse** [comment]
> I think we have some overlap here with #2232, e.g. with pvlib/pvsystem.py

Right. I'll sort that.

File: docs/sphinx/source/whatsnew/v0.11.2.rst [hunk 1]
```diff
@@ -47,6 +47,9 @@ Documentation
   page, using the sphinx glossary directive. (:issue:`1421`, :pull:`2234`)
 * Explained how to write docstrings for new functions in :ref:`example-docstring`
   (:discussion:`2081`, :pull:`2254`)
+* Clarify mounting cases for parameters for :py:func:`~pvlib.temperature.sapm_module`
+  :py:func:`~pvlib.temperature.sapm_cell` and :py:func:`~pvlib.temperature.pvsyst_cell`.
+   (:issue:`1323`, :pull:`2293`)
```
Comment (kandersolar) [inline docs/sphinx/source/whatsnew/v0.11.2.rst:52]: The HTML rendering is a bit messed up here, but let's just fix it along with the other touch ups in the release finalization PR

File: pvlib/pvsystem.py [hunk 2]
```diff
@@ -168,9 +168,11 @@ class PVSystem:
         Inverter parameters as defined by the SAPM, CEC, or other.
 
     racking_model : string, optional
-        Valid strings are 'open_rack', 'close_mount', 'freestanding',
-        'insulated', or 'insulated_back'.
-        Used to identify a parameter set for the cell temperature model.
+        Valid strings are ``'open_rack'``, ``'close_mount'``,
+        ``'freestanding'``, and ``'insulated_back'`. Used to identify a
```
Comment (kandersolar) [inline pvlib/pvsystem.py:172]: Looks like `insulated` got dropped in these descriptions.

File: pvlib/pvsystem.py [hunk 3]
```diff
@@ -168,9 +168,11 @@ class PVSystem:
...
+        parameter set for the SAPM cell temperature model.
```
Comment (kandersolar) [inline pvlib/pvsystem.py:173]: Should this say "SAPM or PVsyst"?

File: pvlib/temperature.py [hunk 4]
```diff
@@ -125,6 +125,18 @@ def sapm_cell(poa_global, temp_air, wind_speed, a, b, deltaT,
...
     +---------------+----------------+-------+---------+---------------------+
 
+    Mounting cases can be described in terms of air flow across and around the
+    rear-facing surface of the module:
+
+    * "open rack" refers to mounting that allows relatively free air flow.
+      This case is typical of ground-mounted systems on fixed racking or
+      single axis trackers.
+    * "close roof" refers to limited or restricted air flow. This case is
+      typical of roof-mounted systems with some gap behind the module.
```
Comment (kandersolar) [inline pvlib/temperature.py:135]: We have a naming inconsistency: `close_mount` in `TEMPERATURE_MODEL_PARAMETERS` but `close_roof` here.  The original report calls it `close roof mount`.  I think we should change the docs to match the code.

File: pvlib/temperature.py [hunk 5]
```diff
@@ -360,6 +396,16 @@ def pvsyst_cell(poa_global, temp_air, wind_speed=1.0, u_c=29.0, u_v=0.0,
     | insulated    | 15.0          | 0.0           |
     +--------------+---------------+---------------+
 
+    Mounting cases can be described in terms of air flow across and around the
+    rear-facing surface of the module:
+
+    * "freestanding" refers to mounting that allows relatively free air
+      circulation around the modules. This case is typical of ground-mounted
+      systems on tilted, fixed racking or single axis trackers.
+    * "insluated" refers to mounting with air flow across only the front
```
Comment (kandersolar) [inline pvlib/temperature.py:405]:
```suggestion
    * "insulated" refers to mounting with air flow across only the front
```

File: pvlib/pvsystem.py [hunk 6]
```diff
@@ -168,9 +168,12 @@ class PVSystem:
         Inverter parameters as defined by the SAPM, CEC, or other.
 
     racking_model : string, optional
-        Valid strings are 'open_rack', 'close_mount', 'freestanding',
-        'insulated', or 'insulated_back'.
-        Used to identify a parameter set for the cell temperature model.
+        Valid strings are ``'open_rack'``, ``'close_mount'``,
+        ``'insulated_back'`, ``'freestanding'`` and ``'insulated'``.
```
Comment (kandersolar) [inline pvlib/pvsystem.py:172]:
```suggestion
        ``'insulated_back'``, ``'freestanding'`` and ``'insulated'``.
```

File: pvlib/pvsystem.py [hunk 7]
```diff
@@ -1393,9 +1396,12 @@ class FixedMount(AbstractMount):
         West=270. [degrees]
 
     racking_model : str, optional
-        Valid strings are 'open_rack', 'close_mount', 'freestanding',
-        'insulated', or 'insulated_back'.
-        Used to identify a parameter set for the cell temperature model.
+        Valid strings are ``'open_rack'``, ``'close_mount'``,
+        ``'insulated_back'`, ``'freestanding'`` and ``'insulated'``.
```
Comment (kandersolar) [inline pvlib/pvsystem.py:1400]:
```suggestion
        ``'insulated_back'``, ``'freestanding'`` and ``'insulated'``.
```

File: pvlib/pvsystem.py [hunk 8]
```diff
@@ -1471,9 +1477,13 @@ class SingleAxisTrackerMount(AbstractMount):
         `cross_axis_tilt`. [degrees]
 
     racking_model : str, optional
-        Valid strings are 'open_rack', 'close_mount', 'freestanding',
-        'insulated', or 'insulated_back'.
-        Used to identify a parameter set for the cell temperature model.
+        Valid strings are ``'open_rack'``, ``'close_mount'``,
+        ``'insulated_back'`, ``'freestanding'`` and ``'insulated'``.
```
Comment (kandersolar) [inline pvlib/pvsystem.py:1481]:
```suggestion
        ``'insulated_back'``, ``'freestanding'`` and ``'insulated'``.
```

**Review by kandersolar — APPROVED** [review]
A few more backticks needed, otherwise LGTM

**Pull request merged**